src/main/java/duke/task/Event.java

```diff
@@ -0,0 +1,41 @@
+package duke.task;
+
+/**
+ * Represents an event task with from and to date.
+ */
+public class Event extends Task {
+
+    protected String from;
+
+    protected String to;
```

Comment:
extremely minor issue, do you think it will be better if there is no space between them? since the rest of your files have no space between the variables